### ISSUE TITLE
Extremely high sharpe

### ISSUE THREAD

**Issue by mathias5986** [freqtrade/freqtrade#4035]
Hello,

I have a sharpe of more than 500. Is this representative or is there something wrong with my config file ? I have a trailing_stop_positive of 0.0000000001. Maybe I shouldn't continue with this strategy and optimize the trailing stop too to get more realistic results?

Thanks,
Mathias

**Comment by xmatthias** [comment]
the sharpe-ratio (as well as sortino) you get from hyperopt cannot really be used to be compared to other sharpe-ratios - it's there as "objective function" which is minimized - so it'll only make sense within the hyperopt context.
We'll also take the negative of it so it can be minimized instead of optimized, to align all loss functions.

The sharpe-ratio fo the "SharpeRatioDaily" might be used for more generic comparisons - but there's  multiple calculation methods / formulaes around which use slightly different approaches, so comparing it to something else is not really the scope of the sharpeRatio's we use for hyperopt.


Apart from that - your trailing-stop-positive is basically `0.0`. I know hyperopt gives you multiple decimals - but overfitting is a big topic with hyperopt - and whenever i see a strategy that uses a value with 5 decimals (or more) i know it's overfitted and will most likely underperform in paper-trading.

**Comment by mathias5986** [comment]
Yes, it looks that the sortino is more defensive (only downside risk), so I will use that one.
It is also probably a better choice to optimize the trailing too (--space roi stoploss trailing).

**Comment by mathias5986** [comment]
Can you also have overfitting if you just optimize roi stoploss trailing in the original strategy file (freqtrade-strategies)? So without changing the already given numbers on the buy and sell side?

**Comment by mathias5986** [comment]
I think the answer is 'yes'. Perhaps I should just continue with the original strategy 😀

**Comment by xmatthias** [comment]
you can overfit everything eventually.
It's however up to you to determine which seems sensible, and which looks like overfitting. Usually, you can also see this in your trade results - if you got only 2 trades, both have a very high profit - it's very likely overfit.
